frontend/src/app/modules/home/home-routing.module.ts

```diff
@@ -18,6 +19,10 @@ const routes: Routes = [{
         path: 'projects',
         component: ProjectsComponent,
     }, {
+        path: 'projects/create-project',
```

Comment:
Path: 'projects/create' would be enough

Reply:
It's bug, because now it look ok

Reply:
![image](https://user-images.githubusercontent.com/36418695/128046685-ffff0a69-7d5a-41e2-b009-18a268c208ee.png)
![image](https://user-images.githubusercontent.com/36418695/128046727-cc6207c1-931a-4395-8981-84cfad5183d4.png)

Reply:
I make screenshot before marge conflict, maybe that's why